Sample Applications/WPFGallery/MainWindow.xaml

```diff
@@ -205,7 +205,8 @@
                     Margin="8,24,0,0"
                     AutomationProperties.Name="Navigation Pane"
                     ItemsSource="{Binding ViewModel.Controls}"
-                    SelectedItemChanged="ControlsList_SelectedItemChanged">
+                    PreviewKeyDown="ControlsList_PreviewKeyDown"
+                    PreviewMouseLeftButtonUp="ControlsList_PreviewMouseLeftButtonUp">
```

Comment:
Have we tested all the navigation scenarios ?
1. Pressing back button
2. Navigation using All Samples page should reflect the changes in the tree view selected item as well
3. The basic ones - pressing enter on the menu item results in navigation
4. Hitting mouse left key results in navigation, whereas nothing happens on pressing right key ?

Reply:
Tested 2,3&4. Added a commit for 1.